atlas/templates/helm/values.yaml.gotmpl

```diff
@@ -82,7 +82,7 @@ db:
   database: {{ .Helm.GetName }}
   sslMode: disable
   port: "5432"
-  serviceName: {{ .Helm.GetName }}-postgres
+  server: {{ .Helm.GetName }}-postgres
```

Comment:
Sorry, I can't find where `.Values.db.server` or `.Values.db.serverName` is used, so I'm having trouble reviewing this. Can you link to the reference so I can confirm it is correct?

Reply:
Sure, [here](https://github.com/infobloxopen/atlas-cli/blob/master/atlas/templates/helm/templates/deployment.yaml.gotmpl#L58) and [here](https://github.com/infobloxopen/atlas-cli/blob/master/atlas/templates/helm/templates/deployment.yaml.gotmpl#L58). I broke it with my previous PR [here](https://github.com/infobloxopen/atlas-cli/pull/137/files#diff-7dba9d433bc2eb55c0f7f255cfceb7ce05001e20fb30c878507fc4cf803d7a93R85)